usr/local/www/diag_sockets.php

```diff
@@ -81,61 +77,78 @@
 		$elements = ($tabindex == 0 ? 7 : 7);
 		$name = ($tabindex == 0 ? 'IPv4' : 'IPv6');
 ?>
-<a name="<?=$name;?>"></a>
-<table style="padding-top:0px; padding-bottom:0px; padding-left:0px; padding-right:0px" width="100%" border="0" cellpadding="0" cellspacing="0" summary="tab">
-<tr><td class="listtopic" colspan="<?=$elements?>"><strong><?=$name;?></strong></td></tr>
-<tr><td>
-<table class="tabcont sortable" id="sortabletable" width="100%" cellspacing="0" cellpadding="6" border="0" summary="results">
+
+		<div class="table table-responsive">
+
+			<table class="table table-hover table-striped table-condensed" id="<?=$name?>">
```

Comment:
This id isn't used anywhere. Could you also remove the redundant newlines around the ```div```?